src/github.com/stellar/horizon/actions_friendbot.go

```diff
@@ -18,7 +18,7 @@ func (action *FriendbotAction) JSON() {
 		action.CheckEnabled,
 		action.LoadAddress,
 		action.LoadResult,
-		action.LoadResource,
+		action.loadResource,
```

Comment:
Not sure what the desired Go naming is, but the `loadResource` is different compared to the other `action`s